ci-operator/step-registry/cilium/conf/cilium-conf-ref.metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+	"path": "cilium/conf/cilium-conf-ref.yaml",
```

Comment:
Mix of tabs and spaces.

Use `make update` to regenerate these files

Reply:
Ah, didn't realise these were generated... thanks!